runelite-client/src/main/java/net/runelite/client/ui/components/IconTextField.java

```diff
@@ -169,6 +169,20 @@ public void changedUpdate(DocumentEvent e)
 		add(clearButton, BorderLayout.EAST);
 	}
 
+	@Override
+	public void addMouseListener(MouseListener mouseListener)
+	{
+		super.addMouseListener(mouseListener);
```

Comment:
should this really apply to whole icon text field? and not just text field and icon?